week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -84,7 +87,13 @@ function getTransportModes() { }
 
   Hint: Use the corresponding array method to decide if an element is member of an array.
 */
-function isAccessibleByTransportMode() { }
+function isAccessibleByTransportMode(mode, transport) {
+  if (mode.includes(transport)) {
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
This works but don't forget that 'includes' returns either true or false, so a better way would be:
return mode.includes(transport);